force-app/main/default/classes/TestDataFactory.cls

```diff
@@ -0,0 +1,58 @@
+public with sharing class TestDataFactory {
+    private static String DEFAULT_STATUS_VALUE = 'Contracted';
+    // public TestDataFactory() {
```

Comment:
You should probably delete this commented out code

Reply:
Thanks, @aidan-harding! Yes..better to remove. Cheers.